README.md

```diff
@@ -32,9 +34,92 @@ For alternative ways, stop by our Community Hub [stackrox.io](https://www.stackr
+  --set central.resources.requests.memory=1Gi \
+  --set central.resources.requests.cpu=1 \
+  --set central.resources.limits.memory=4Gi \
+  --set central.resources.limits.cpu=1 \
+  --set scanner.autoscaling.disable=true \
+  --set scanner.replicas=1 \
+  --set scanner.resources.requests.memory=500Mi \
+  --set scanner.resources.requests.cpu=500m \
+  --set scanner.resources.limits.memory=2500Mi \
+  --set scanner.resources.limits.cpu=2000m
+```
+To create a secured cluster, you first need to generate an init bundle containing initialization secrets. The init bundle will be saved in `stackrox-init-bundle.yaml`, and you will use it to provision secured clusters as shown below.
+```sh
+kubectl -n stackrox exec deploy/central -- roxctl --insecure-skip-tls-verify \
+  --password "$(cat stackrox-admin-password.txt)" \
+  central init-bundles generate stackrox-init-bundle --output - > stackrox-init-bundle.yaml
+```
+Set a meaningful cluster name for your secured cluster in the `CLUSTER_NAME` environment variable. The cluster will be identified by this name in the clusters list of the StackRox UI.
+```sh
+CLUSTER_NAME="my-secured-cluster"
+```
+Then install stackrox-secured-cluster-services (with the init bundle you generated earlier) using this command:
+```sh
+helm install -n stackrox stackrox-secured-cluster-services stackrox/stackrox-secured-cluster-services \
+  -f stackrox-init-bundle.yaml \
+  --set clusterName="$CLUSTER_NAME"
+```
+When deploying stackrox-secured-cluster-services on a different cluster than the one where stackrox-central-services are deployed, you will also need to specify the endpoint (address and port number) of Central via `--set centralEndpoint=<endpoint_of_central_service>` command-line argument.
+
+When deploying StackRox on a small node, you can install with additional options. This should reduce stackrox-secured-cluster-services resource requirements. Please keep in mind that these reduced resource settings are not recommended for a production setup.
+```sh
+helm install -n stackrox stackrox-secured-cluster-services stackrox/stackrox-secured-cluster-services \
+  -f stackrox-init-bundle.yaml \
+  --set clusterName="$CLUSTER_NAME" \
+  --set sensor.resources.requests.memory=500Mi \
+  --set sensor.resources.requests.cpu=500m \
+  --set sensor.resources.limits.memory=500Mi \
+  --set sensor.resources.limits.cpu=500m
+```
+
+To further customize your Helm installation consult these documents:
+* <https://docs.openshift.com/acs/installing/installing_helm/install-helm-quick.html>
+* <https://docs.openshift.com/acs/installing/installing_helm/install-helm-customization.html>
+
+### First use
+
+Follow these steps to get to StackRox UI
+
+1.Setup port forward to central:
+```
+kubectl -n stackrox port-forward deploy/central 8000:8443
+```
+
+2.Open <https://localhost:8000> in your browser.
+
+3.Accept certificate warnings and proceed.
+
+4.Log in as `admin` using the password in `stackrox-admin-password.txt`
```

Comment:
do we need to tell users where this file is? (txt file)

Reply:
Yes, we specifically generate this password file to make it more convenient for the user. The previous approach was to not specify a password and extract the automatically generated password using `kubectl -n stackrox get secret central-htpasswd -o go-template='{{index .data "password" | base64decode}}'`

But it turns out only Operator-based install write the `.data.password` attribute, so we decided to manually set a password instead. It's written into `stackrox-admin-password.txt` to be easily accessible, users are free to change that of course.